opalog/policy.go

```diff
@@ -126,6 +126,29 @@ func (rule *Rule) Equal(other *Rule) bool {
 	return rule.Body.Equal(other.Body)
 }
 
+// DocKind represents the collection of document types that can be produced by rules.
+type DocKind int
+
+const (
+	// ScalarDoc represents a scalar document type.
+	ScalarDoc = iota
```

Comment:
These enums are poorly named. The intention is to identify whether the rule produces a completely defined document or a partially defined set/object.

In the next set of changes I have renamed them accordingly.